frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -26,6 +26,14 @@ import { teamLogic } from '../teamLogic'
 import { urls } from 'scenes/urls'
 import { getInsightId } from 'scenes/insights/utils'
 
+export const BREAKPOINTS: Record<DashboardLayoutSize, number> = {
+    sm: 1024,
+    xs: 0,
+}
+export const COLS: Record<DashboardLayoutSize, number> = { sm: 12, xs: 1 }
+export const MIN_W = 4
+export const MIN_H = 6
```

Comment:
I'd expand columns, width, and height instead of abbreviating. But it's only a nit since they're clear in context